pubtools/_quay/manifest_claims_handler.py

```diff
@@ -0,0 +1,210 @@
+# this code has been copied from rcm-pub (pubd/lib/docker_signature.py)
```

Comment:
Is there a plan to avoid code duplication? I remember some discussion but would be nice to have at least some FIXME or, even better, a Jira reference here.

Reply:
In this particular scenario I felt that it would be easiest to just copy-paste the code from pub, since there's a lot of opaque logic happening in this class. I imagine we'll do a major cleanup of pub code after Dynamo is up and running.